client/src/App.js

```diff
@@ -3,6 +3,7 @@ import React, { Component } from 'react';
 import SavedList from './Movies/SavedList';
 import MovieList from './Movies/MovieList';
 import Movie from './Movies/Movie';
+import Route from 'react-router-dom/Route';
```

Comment:
Thanks for pushing the recent changes. And good work on your progress so far! 👍 👍 👍  Don't think this will be  an issue since it could be imported in other ways, but for future reference you can keep the `import Module from 'module';` format to show its a dependency and not an actual path in the client directory.